examples/Reporting/ParallelReportDownload.cs

```diff
@@ -0,0 +1,242 @@
+// Copyright 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Services;
+using static Google.Ads.GoogleAds.V3.Services.GoogleAdsServiceClient;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// Shows how to download a set of reports from a list of accounts in parallel. If you need
+    /// to obtain a list of accounts, please see the GetAccountHierarchy or
+    /// ListAccessibleCustomers examples.";
+    /// </summary>
+    public class ParallelReportDownload : ExampleBase
+    {
+        // Defines the Google Ads Query Language (GAQL) query strings to run for each customer ID.
+        private readonly string[] GAQL_QUERY_STRINGS =
+        {
+            @"SELECT campaign.id, metrics.impressions, metrics.clicks
+                FROM campaign
+                WHERE segments.date DURING LAST_30_DAYS",
+            @"SELECT campaign.id, ad_group.id, metrics.impressions, metrics.clicks
+                FROM ad_group
+                WHERE segments.date DURING LAST_30_DAYS"
+        };
+
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            ParallelReportDownload codeExample = new ParallelReportDownload();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer IDs for which the call is made.
+            string customerIds = "INSERT_CUSTOMER_ID_1_HERE, INSERT_CUSTOMER_ID_2_HERE";
```

Comment:
Make this a long[] array.

long[] customerIds = new long[] {
   long.Parse("INSERT_CUSTOMER_ID_1_HERE"),
   long.Parse("INSERT_CUSTOMER_ID_2_HERE")
}

and then pass the array to the Run() method.

Reply:
Done.

Reply:
Done.